### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#335] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-23 104015](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/c1703d22-c926-4963-b189-e4ad22b9b286)

**Comment by Elia-Belli** [comment]
- **(1) Falso**, ad esempio $(1,0,0,0,0,0),(0,1,0,0,0,0),(0,0,1,0,0,0),(0,0,0,1,0,0)$ sono $4$ vettori linearmente indipendenti in $\mathbb{R}^6$.
-  **(2) Vero**, sappiamo che una base di $\mathbb{R}^4$ ha cardinalità $4$ ed è l'insieme massimale di vettori linearmente indipendenti, allora aggiungendo qualsiasi altro vettore si otterrà sempre dipendenza lineare.
- **(3) Falso**, ad esempio $(1,0,0,0,0,0),(2,0,0,0,0,0),(3,0,0,0,0,0),(4,0,0,0,0,0)$ sono linearmente dipendenti in $\mathbb{R}^6$ (perchè proporzionali)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/7ed10d39-d206-4e5e-b2e8-afa2829663c0)

**Comment by CiottoloMaggico** [comment]
I vettori "e" sono vettori appartenenti alla base canonica
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/61b23292-5fa8-4e5e-ae28-9ab44d66c0e6)
